devenv/README.md

```diff
@@ -2,11 +2,43 @@
 
 A Developer Environment (or devenv for short) is a workspace for developers to make changes without breaking anything in a live environment. 
 
-In order to foster contribution for the Netdata Agent and it's various collectors, we created an image that is a devenv on it'own, a fully containerized environment where it has everything a developer needs to develop improvements on the Netdata Agent itself or it's collectors. 
+In order to foster contribution for the Netdata Agent and it's various collectors, we created an image that is a devenv on it's own, a fully containerized environment where it has everything a developer needs to develop improvements on the Netdata Agent itself or it's collectors.
 
-# Instructions 
+We have included all the dependencies that you need to build the Netdata Agent and it's collectors. Moreover, we have included helpful tools such as `python pip` and `flake8`. 
```

Comment:
I would make this second sentence something like: We have additionally included many of the tools we use for checking PRs, including various linters like `flake8` and `shellcheck`’.

We want to point out that the extra stuff we included is actively useful to the development process we use, not that they’re just generically useful.